CreateCarthageArchive.sh

```diff
@@ -1,6 +1,6 @@
-# carthage archive | tee "carthageout.txt"
-frameworkfilename=$(carthage archive |  grep -o 'Created .*\.zip$'  | grep -o   '\S*\.zip$')
-
+carthage archive | tee "carthageout.txt"
+ls *.zip
```

Comment:
why are you running "ls" here?

Reply:
I believe this is for debugging purposes.